package-lock.json

```diff
@@ -643,6 +643,24 @@
       "integrity": "sha512-JoX0apGbHaUJBNl6yF+p6JAFYZ666/hhCGKN5t9QFjbJQKUU/g8MNbFDbvfrgKXvI1QpZplPOnwIo99lX/AAmA==",
       "dev": true
     },
+    "ansi-escapes": {
```

Comment:
Are all these libraries being used? Not sure why there's so much being added to `package-lock.json`. I'd try reverting these changes and verifying everything works as expected, and if that's the case this can be reverted

Reply:
I think in the old PR 'lint' was added as a dependency by accident, I think that caused `package-lock.json` to add these additional ~800 lines. It should be fine if we revert it.

Reply:
LOL that's right